### DISCUSSION TITLE
Save data.

### DISCUSSION THREAD

**Discussion by cenimm** [mupen64plus-ae/mupen64plus-ae#1064] [General]
I have save data that's is .srm but cant load it why?
Can I convert it somehow on my android phone?

**Comment by fzurita** [comment]
The app doesn't support the SRM format. Is that from RetroArch?

**Reply by cenimm** [reply]
I don't really remember. I have old saves on my onedrive that I send to my phone but got stuck when I tryed to load it.
Is it possible to convert it on my phone or is my save dead?

**Reply by fzurita** [reply]
Yeah, I'm not sure I can help with that. Maybe it's possible, but I don't know if any tools to do it for you.
